example/02-kademlia/rendezvous_chat.cpp

```diff
@@ -42,7 +42,7 @@ class Session : public std::enable_shared_from_this<Session> {
     }
 
     stream_->readSome(
-        gsl::span(incoming_->data(), static_cast<ssize_t>(incoming_->size())),
+        std::span(incoming_->data(), static_cast<ssize_t>(incoming_->size())),
```

Comment:
Remove clang-tidy casts for `ptrdiff_t`/`ssize_t` used by `gsl::span`
```suggestion
        std::span(incoming_->data(), incoming_->size()),
```